docs/pages/index.tsx

```diff
@@ -297,6 +298,12 @@ export function JoinTheCommunity() {
             icon={<RedditIcon className="icon-lg text-palette-white" />}
             iconBackground="#FC471E"
           />
+          <CommunityGridCell
+            title="Newsletter"
+            description="Get the latest updates from monthly Expo newsletter."
+            link="http://eepurl.com/hk1tCn"
+            icon={<Mail01DuotoneIcon className="icon-lg text-palette-white" />}
```

Comment:
```suggestion
            icon={<Mail01DuotoneIcon className="icon-lg text-palette-white" />}
```

Similar here, we should use outline or solid variation. Duotone variant is mainly reserved for large icons or marketing usages.

Reply:
I've switch to normal variant, wdyt?

<img width="569" alt="CleanShot 2023-09-18 at 11 35 24@2x" src="https://github.com/expo/expo/assets/10234615/ec4119b4-29ce-4c11-8b34-523e76a0adaa">